src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/deployment/pipeline_management.yml

```diff
@@ -523,7 +523,6 @@ Resources:
               runtime-versions:
                 python: 3.9
                 nodejs: 14
-            pre_build:
```

Comment:
Why has this been removed?

Reply:
As the next lines are installation steps, not pre-build commands.
This has always been a `pre-build` step before, so not something that was introduced incorrectly.
I just figured it makes more sense to put it in the install step instead, as that is what it is doing.